frontend/pages/media/podcast.js

```diff
@@ -237,15 +238,18 @@ const Podcast = () => {
 
           <div id={styles.episodes}>
             {selectedPosts.map((episode, index) => {
+              const { episodeNo, title, date, description } = episode.fields;
+              date = formatPodcastDate(date);
+              const { url } = episode.fields.img.fields.file;
               return (
                 <EpisodeTemplate
                   key={index}
-                  episodeNo={episode.episodeNo}
-                  title={episode.title}
-                  cover={episode.imgUrl}
-                  date={moment(episode.date).format("MMMM DD, YYYY")}
-                  description={episode.description}
-                  episode={episode}
+                  episodeNo={episodeNo}
+                  title={title}
+                  cover={url}
+                  date={date}
```

Comment:
Can we define the fields in the EpisodeTemplate.js file rather than defining them here e.g. {title, cover} and passing it in?